docs/resources/artifactory_push_replication.md

```diff
@@ -58,6 +58,7 @@ The following arguments are supported:
     * `sync_properties` - (Optional)
     * `sync_statistics` - (Optional)
     * `path_prefix` - (Optional)
+    * `proxy` - (Optional)
```

Comment:
Needs a description - preferrably something from official documentation @alexhung - where have you been getting yours?

Reply:
I couldn't find any official description so I made this one up: https://github.com/jfrog/terraform-provider-artifactory/blob/master/docs/resources/artifactory_artifact_webhook.md?plain=1#L53

Reply:
Also after having worked with proxy in couple of different scenarios now, I think we should call this attribute `proxy_key` which is more descriptive since this isn't the proxy hostname.

Reply:
I'll add a description (probably copying https://github.com/jfrog/terraform-provider-artifactory/blob/master/docs/resources/artifactory_artifact_webhook.md?plain=1#L53).

It's super confusing because the GET API for replications returns back the field named `proxy_ref` but the CREATE/UPDATE takes a field named `proxy`.

Since https://github.com/jfrog/terraform-provider-artifactory/blob/master/docs/resources/artifactory_artifact_webhook.md?plain=1#L53 I'm slightly in favor of keeping it as `proxy` for the sake of consistency.  If you guys feel strongly about renaming this to `proxy_ref`, I can do that

Reply:
@Guent4 Let's keep this as `proxy` for consistency, as you mentioned.